ocaml/parsing/parsetree.mli

```diff
@@ -1026,6 +1028,7 @@ and open_declaration = module_expr open_infos
 
 and 'a include_infos =
     {
+     pincl_kind : include_kind;
```

Comment:
I think in a perfect world we would probably make `Psig_include_functor` and `Pstr_include_functor` their own things, and then do similarly in typed tree.  That would make it harder to write bugs that conflate `include` with `include functor`.  But considering typed tree already does it this way (my fault) this is fine, just an idle remark.